src/routes/(console)/project-[region]-[project]/messaging/message-[message]/updateTopics.svelte

```diff
@@ -139,17 +140,21 @@
             {:else if message.status === 'draft'}
                 <Empty on:click={() => (showTopics = true)}>Add a topic</Empty>
             {:else}
-                <EmptySearch hidePagination>
-                    <div class="u-text-center">
-                        No topics have been selected.
-                        <p>
-                            Need a hand? Check out our <Button
-                                href="https://appwrite.io/docs/products/messaging/topics">
-                                documentation</Button
-                            >.
-                        </p>
-                    </div>
-                </EmptySearch>
+                <Card.Base padding="none">
+                    <PinkEmpty
+                        type="secondary"
+                        title="No topics have been selected"
+                        description="Need a hand? Check out our documentation.">
```

Comment:
if not a draft, means the message either succeeded, or failed. We should have the copy checkin in past tense imo. Not sure if we need docs link for that as well but that fine. The message copy needs an update ig.